### ISSUE TITLE
Pure Pool (300610)

### ISSUE THREAD

**Issue by furycd001** [ValveSoftware/Proton#2352]
# Compatibility Report
- Name of the game with compatibility issues: Pure Pool
- Steam AppID of the game: 300610

## System Information
- GPU: NVIDIA Corporation GeForce 940MX
- Driver/LLVM version: 4.5.0 NVIDIA 384.130
- Kernel version: 4.15.0-45-generic
- Link to full system information report: https://gist.github.com/furycd001/9a84c88f312a674ada56dca03cfe8e9b
- Proton version: 3.7-8 & 3.16-4 & 3.16-7

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

https://paste.ubuntu.com/p/SdSyxrfds7/
`<removed by moderator>`

## Symptoms:
The game refuses to launch and simply does nothing. Whenever you click on play, it will say that the game is running for about 2 seconds & then switch to saying syncing for around another 2 seconds. Meanwhile nothing opens on screen.

## Reproduction:
Simply try to launch the game....

**Comment by kisak-valve** [comment]
`err:mscoree:CLRRuntimeInfo_GetRuntimeHost Wine Mono is not installed` in the log points towards #18.

**Comment by IanTrudel** [comment]
Found a solution! I installed dotnet48 and d3dcompiler_47 using protontricks and ran `<Link removed by moderator>` on the prefix. Graphics are fantastic and fast, audio and DS4 controller work.

```
protontricks 300610 --force -q dotnet48 d3dcompiler_47
<Removed by moderator>
```

![image](https://user-images.githubusercontent.com/8844578/70281257-95890f00-1788-11ea-8ee4-4d85b5dd9bac.png)

![image](https://user-images.githubusercontent.com/8844578/70281458-10522a00-1789-11ea-85be-44ae33e83178.png)

![image](https://user-images.githubusercontent.com/8844578/70281397-f1539800-1788-11ea-815c-956e265b42b5.png)

![image](https://user-images.githubusercontent.com/8844578/70281416-fa446980-1788-11ea-9449-216b11028073.png)

![image](https://user-images.githubusercontent.com/8844578/70281427-003a4a80-1789-11ea-81ec-c595ecb81ed3.png)

![image](https://user-images.githubusercontent.com/8844578/70281439-04fefe80-1789-11ea-9a41-1829c1f5c433.png)

![image](https://user-images.githubusercontent.com/8844578/70281448-0af4df80-1789-11ea-8fce-690096a47c43.png)

**Comment by kisak-valve** [comment]
Hello @BackOrder, a workaround you posted is legally problematic and has been removed.

**Comment by IanTrudel** [comment]
> Hello @BackOrder, a workaround you posted is legally problematic and has been removed.

Oh? My apologies, I'm not aware of this. What is the problem exactly?

**Comment by kisak-valve** [comment]
Libraries from Windows are copyrighted and we can not condone the distribution of them by third parties.

**Comment by IanTrudel** [comment]
> Libraries from Windows are copyrighted and can not be distributed by third parties.

Yes, duly noticed. I deleted the prefix to try again with protontricks only and turns out the `<moderator is watching me>` is not necessary.

`protontricks 300610 --force -q dotnet48 d3dcompiler_47` will do! Cheers. :)
